README.md

```diff
@@ -11,3 +11,9 @@ cd app
 bun install  # npm, yarn, pnpm and other node package managers should work too
 bun run dev
 ```
+
+On commit, the app is built and deployed to Vercel.
+
+On a non-main branch, When a build is successful it is deployed to a preview version: https://badminton-j1sedcfjc-jake-walkers-projects.vercel.app
```

Comment:
The link here is for a specific commit, people should follow the link from the Vercel bot in pull requests, etc... for their branch

Reply:
Ahh, you didn't tell me that :P